packages/taco/test/conditions/context.test.ts

```diff
@@ -262,15 +262,21 @@ describe('param or context param schema', () => {
   });
 
   it('accepts a hex string', () => {
-    expect(paramOrContextParamSchema.safeParse("deadbeef").success).toBe(true);
+    expect(paramOrContextParamSchema.safeParse('deadbeef').success).toBe(true);
```

Comment:
Careful here - any hex string as a representation of bytes should be prefixed by `0x`. Strings that are just strings don't need the `0x` prefix.